css/responsive/laptop.styl

```diff
@@ -177,6 +177,24 @@ $footer-height = 53px
   font-size: 24px
   
 // ******************************************************
+
+// HEADER
+
+.header-logo-mobile
+  display: none
+
+.header-logo-desktop
+  display: initial
+  
+.header-logo 
+  svg 
```

Comment:
ultimately I would say this either should just be a global svg override to go with the img override, or the svg should have this ID